docs/source/workflows.rst

```diff
@@ -813,6 +813,13 @@ and it finally performs a minimization for the final fine tuning of the optimum
         "number_of_molecules": 1
     }
 
+
+  In particular:
+
+  * ``host/ads`` describes the interaction between host and (all) adsorbates, ``ads/ads`` the interaction between adsorbates
+  * The binding energy is the final value of ``energy_host/ads_coulomb_final`` (you still need to divide by ``number_of_molecules``)
+  * ``energy_ads/ads_coulomb_final`` and ``energy_ads/ads_tot_final`` may be non-zero even for a single molecule due to rounding errors in Ewald summation
```

Comment:
and ``energy_ads/ads_vdw_final``

Reply:
well, the van-der-waals energies are usually computed with cutoffs, thus they should be exactly zero for a single molecule (which they are in your example as well)

Reply:
> (you still need to divide by ``number_of_molecules``)
It depends what you want to get, I wouldn't write it as a mandatory sentence
I would write something as:
> In the case of ``number_of_molecules``>1, you may want to obtain an average binding energy per molecule by dividing  this value by the number of molecules.